variables.tf

```diff
@@ -6,6 +6,10 @@ variable "ssh_public_key" {
   type = string
 }
 
+variable "ssh_user" {
+  type = string
```

Comment:
Docs say it's optional, but there is no default value.

Reply:
I agree it should have a default here as well, but it is mentioned in the readme inside the example tfvars file

Reply:
> and optionally `ssh_user` if you are using a different distribution

Reply:
Given that we don't have a default image, it probably doesn't make sense to have a default SSH user

Reply:
Changed it in the docs